doc/manual/manual.bib

```diff
@@ -12,6 +12,18 @@ @string{ajs
 @string{cmame = {Computer Methods in Applied Mechanics and Engineering}}
 @string{pnas = {Proceedings of the National Academy of Sciences}}
 
+@Article{THIE17,
+AUTHOR = {Thieulot, C.},
+TITLE = {Analytical solution for viscous incompressible \hack{\break}Stokes flow in a spherical shell},
```

Comment:
what do these latex commands do? can we remove them?

Reply:
I guess so. I downloaded this bibtex entry from the journal.